packages/react-sdk/src/components/BoardBar/ExportWhiteboardDialog.test.tsx

```diff
@@ -170,9 +185,9 @@ describe('<ExportWhiteboardDialog/>', () => {
   });
 
   it('should handle error while generating PDF', async () => {
-    jest
-      .mocked(createWhiteboardPdf)
-      .mockReturnValue(throwError(() => new Error('Failed')));
+    vi.spyOn(pdf, 'createWhiteboardPdf').mockReturnValue(
```

Comment:
```suggestion
    vi.mocked(pdf, 'createWhiteboardPdf').mockReturnValue(
```

Reply:
This does not work and makes the test fail if I do that.